Example/PrebidDemo/PrebidDemoSwift/InstreamVideoViewController.swift

```diff
@@ -0,0 +1,148 @@
+/*   Copyright 2018-2019 Prebid.org, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+import UIKit
+import GoogleInteractiveMediaAds
+import PrebidMobile
+
+class InstreamVideoViewController: UIViewController, IMAAdsLoaderDelegate, IMAAdsManagerDelegate {
```

Comment:
Can we add a video player with a real video

Reply:
This is preferred to showcase to real in-stream integration for publishers to follow. Either present the video ad before the video content or in the middle of content, passing the appropriate signal such as pre-roll or mid-roll or post-roll.

Reply:
Updated with the content play logic in sample app